pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -375,7 +390,10 @@ func (r *ReconcileWindowsMachine) Reconcile(request reconcile.Request) (reconcil
 	}
 	r.recorder.Eventf(machine, core.EventTypeNormal, "MachineSetup",
 		"Machine %s configured successfully", machine.Name)
-
+	// configure prometheus if it is not configured
```

Comment:
Again, the comment should explain why. The main issue is that it is not clear why we call it in multiple places. I feel we are being overzealous in calling `pc.Configure()`.